containers/jenkins-master/config/jobs/create-cloud-image-rolling-edge/config.xml

```diff
@@ -25,7 +25,33 @@
       <spec>H/30 * * * *</spec>
       <entries>
         <org.jenkinsci.plugins.urltrigger.URLTriggerEntry>
-          <url>http://system-image.ubuntu.com/ubuntu-core/rolling/edge/generic_amd64/index.json</url>
+          <url><![CDATA[https://search.apps.ubuntu.com/api/v1/search?q=architecture:amd64,name:ubuntu-core.canonical,channel:edge&size=1]]></url>
```

Comment:
> We should start thinking about making images for two stable and one edge

It should be easy for the job configuration, just one trigger instead of two. The image creation can be more difficult but not too much, it should be done adhoc for the test execution and we need to pass the channels for each snap to snappy-cloud-image (in other words, more flags)